ampersand-chart.js

```diff
@@ -822,9 +823,9 @@
       var barMargin = barWidth * b;
 
       var y = d3.scale.linear()
-        .domain(_.result(this.model, 'range', [ 0, d3.max(data, function(d) {
+        .domain(_.result(this.model, 'range', [ 0, Math.max(this.model.yAxisMinimum, d3.max(data, function(d) {
```

Comment:
Can you extract a function here while you're editing this code?  A simple utility function that computes the range for one of these type of graphs.  Something like this would be nice: `domain(this.model.defaultedRange)` where defaultedRange is a derived property on the ChartState:
```js
derived: {
  'defaultedRange': {
    deps: ['yAxisMinimum', '_data', 'values'],
    fn: function() {
      return _.result(this, 'range', [0, Math.max(this.yAxisMinimum, d3.max(this._data.models, function(d) {
        return Math.max.apply(null, _.remove(_.values(_.pick(d.attributes, this.values)), function(n) { return !isNaN(n); }));
      }))
    }
  }
}
```